subprojects/core/src/test/groovy/org/gradle/internal/progress/DefaultBuildOperationExecutorTest.groovy

```diff
@@ -645,4 +647,10 @@ class DefaultBuildOperationExecutorTest extends ConcurrentSpec {
 
         void run(BuildOperationContext buildOperationContext) {}
     }
+
+    ParallelExecutionManager parallelExecutionManager() {
```

Comment:
Heh, I actually thought the same thing and then forgot to do it.  I've captured this in a follow-up story.